### PR TITLE
Update Discord.js dependencies

### PR DESCRIPTION
**Pull request by aedwardg**
## What issue is this solving?

No official issue, but recommended in the dev server here:
https://discord.com/channels/810930683732033597/810930684171780117/981746378424328252

### Description

Updates Discord.js and related dependencies.

## Any helpful knowledge/context for the reviewer?

- Is a re-seeding of the database necessary? :x:
- Any new dependencies to install? :heavy_check_mark:
- Any special requirements to test? Make sure everything still works
  - (e.g., admin perms, alt account, etc.)

### Please make sure you've attempted to meet the following coding standards
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code

### CODE REVIEW

**Comment by mikejoh12** [comment]
The update to builders apparently removed the addChoice method so it is throwing an "addChoice is not a function" error now when running the the bot or deploying the slash-commands.

The method was replaced by addChoices: https://github.com/discordjs/discord.js/blob/main/packages/builders/src/interactions/slashCommands/mixins/ApplicationCommandOptionWithChoicesAndAutocompleteMixin.ts#L26

**Comment by aedwardg** [comment]
Well that's annoying. I figured there would be no breaking changes since these were all minor version upgrades.  I'll make the necessary changes and ping on discord when it's ready for re-review

**Review by mikejoh12 — APPROVED** [review]
LGTM!

File: slash-commands/utility/helpcenter.js [hunk 1]
```diff
@@ -25,21 +25,25 @@ module.exports = {
...
-          'https://help.codecademy.com/hc/en-us'
+        .addFields(
+          {
+            name: 'Codecademy Help Center',
+            value: 'https://help.codecademy.com/hc/en-us',
+          },
+          {
+            name: 'Submit A Request',
+            value: 'https://help.codecademy.com/hc/en-us/requests/new',
+          },
```
Comment (Victoria-DR) [inline slash-commands/utility/helpcenter.js:36]: I believe this was removed for the non-slash version of the command by Fede in the past. It should also be removed for the slash command in both the plaintext and embed versions.

**Review by Victoria-DR — APPROVED** [review]
LGTM! I added a comment on the helpcenter command that isn't related to what this PR is about, but it's a minor change so I think we can implement it here. Approved regardless.

**Pull request merged**